### PR TITLE
Visual treatment for bottom app bar demo

### PR DESCRIPTION
**Pull request by xster**
#16821

- Fix list safe area in demo
- Update UI a bit to reduce crowding

flutter_01

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]

**Comment by HansMuller** [comment]
We've solicited some design input for this demo. Will wait for the new design before landing any changes.

**Comment by xster** [comment]
Closing this for now while waiting for designer feedback

**Pull request closed without merging**
